docs/guides/ExampleApp/backend/release/Dockerfile.api

```diff
@@ -0,0 +1,10 @@
+FROM mcr.microsoft.com/dotnet/aspnet:6.0
```

Comment:
Shouldn't we prefer deployments via Argo nowadays? That would make this entire directory unnecessary, just like "deploy to test" step of the Jenkinsfile.

But if we don't, then this should probably use .NET 7.0 just like dev env images.